selfdrive/car/ford/carcontroller.py

```diff
@@ -69,13 +69,22 @@ def update(self, CC, CS, now_nanos):
 
       self.apply_curvature_last = apply_curvature
 
+      steeringPressed = CS.out.steeringPressed
+      steeringAngleDeg = CS.out.steeringAngleDeg
+
+      if steeringPressed and abs(steeringAngleDeg) > 60:
+        apply_curvature = 0
```

Comment:
This will trip the panda safety causing messages to be dropped and the system will stop applying steering. Have you checked the `totalTxBlocked` in plotjuggler?

Reply:
I have not checked totalTxBlocked, but we are not receiving any of the warnings on screen "steering temporarily unavailable" that we typically see when we run afoul of panda.

Reply:
This just likely prevents wind up while steering at high curvatures. The issue isn't Ford specific, it's a problem with any low-rate angle control car, so I would like to avoid make-specific logic.

Reply:
I tested Cameron's suggestion of disabling lateral during human turns and achieved the same result of eliminating the problem.  I have not experienced this issues in other cars (I have only driven my F150 and a Santa Fe on OP).  But it clearly isn't as severe on other cars or you would not be making a suggestion to close this PR, because it an unacceptable event.  If you wish for me to move this code to a place where it affects all makes and models I am okay with that. If you wish for me to update using lateral active to achieve the result instead of ramp_type and curvature, I'm okay with that too.  I will make a video tonight of how this behaves on stock OP and post it here

Reply:
Here is a route exhibiting the issues with stock OP logic:
24574459dd7fb3e0/2024-02-20--19-24-10/0

Here is a video showing the wheel action for 3 of the turns:
https://youtu.be/6yM6Kt6ibwA

Here is a route with the Cameron's suggest logic to disable lateral during the turn:
24574459dd7fb3e0/2024-02-20--19-44-38/0

Here is the video associated with some turns on that route:
https://youtu.be/TcXg6q8q42o

Since we have already moved away from the idea of using ramp_type, I have not added extraneous data related to Ford's stock logic here.

If this is an issue that affects multiple vehicles, I can cancel this PR and create a new one that makes the logic universal.  If after looking at the video you believe it to be a Ford issue, then I will update the PR with the latest logic to cancel lateral instead of use ramp_type.  I'm good with either one, but the issue addressed by this PR can be ignored.